docs/zh/reference/sql/functions_and_operators/Files/udfs_8h.md

```diff
@@ -656,7 +656,7 @@ Example:
 
 ```sql
 
-select concat("-", "1", 2, 3, 4, 5.6, 7.8, Timestamp(1590115420000L));
+select concat_ws("-", "1", 2, 3, 4, 5.6, 7.8, Timestamp(1590115420000L));
```

Comment:
So docuemnts in default_udf_library is not correct?

Reply:
Fixed too.